docs/alerting-recording.md

```diff
@@ -0,0 +1,59 @@
+# Alerting & Recording Rules
+
+### Alerting Rules
+
+The alerting rules are used to trigger alerts based on the provided alerting rules an alert should be fired to an external service like slack. In promscale the alerting rules are evaluated in prometheus. i.e define alerting rules in prometheus config
+
+```
+groups:
+  - name: <alert-group-name>
+    rules:
+    - alert: <alert-name>
+      expr: <promql_expression>
+      for: <time-interval for how long this to happen to happen to fire an alert>
+      labels:
+        <key>:<value>
+      annotations:
+        summary: <text>
+        description: <description on alert>
+```
+
+Load the above defined alert rule file to your prometheus config for evaluation. i.e 
```

Comment:
Maybe, have this statement linked with the above one after updating.

Reply:
The sequence is to add alerting rules in a file, define a reference to the rules file, and configure the alert manager endpoint.

Reply:
Can we please change `config` -> `configuration` and `i.e` -> `i.e.,` (along with full-stops where needed) across the entire file?